src/Discord/MessageSender.php

```diff
@@ -63,7 +63,11 @@ public function sendSecretMessage(SecretSanta $secretSanta, string $giver, strin
             $precision = null;
 
             if (($response = $e->getResponse()) && 403 === $response->getStatusCode()) {
-                $precision = 'The user does not allow to receive DM on this server. Please ask them to change their server settings.';
+                $precision = sprintf(
+                    '@%s does not allow to receive DM on the server "%s". Please ask them to change their server privacy settings as explained in our faq.',
```

Comment:
link to the FAQ?

Reply:
I don't really want to generate an url here and to include this url in an exception message. On the exception page, there is a paragraph below the displayed error that mention and link to the faq